composer.json

```diff
@@ -6,11 +6,12 @@
 
     , "require": {
           "php": "^5.5|^7.0"
-        , "illuminate/database": "~5.2.0"
-        , "symfony/dependency-injection": "^2.6|^3.0"
+        , "illuminate/database": "^5.0.0 <5.3"
+        , "symfony/dependency-injection": "^2.8|^3.0"
```

Comment:
erm actually not sure about that, but I kinda want to propose to push to Sf 2.8 and PHP 5.6 min, or even Sf 3.0 and PHP7. WDYT?